bors/Cargo.toml

```diff
@@ -17,6 +17,7 @@ hyper-tls = "0.5"
 reqwest = "0.11"
 liquid = "0.21"
 log = "0.4.8"
+lru = "0.7.3"
```

Comment:
I've never seen any duplicate webhooks be sent by github. Are you should you don't have your webhook delivery misconfigured and somehow have multiple webhooks setup to deliver to your instance of bors?

Reply:
I’m pretty sure there’s only one webhook, cause the delivery IDs are the same